src/models/room.js

```diff
@@ -1311,6 +1302,28 @@ Room.prototype.addPendingEvent = function(event, txnId) {
 
     this.emit("Room.localEchoUpdated", event, this, null, null);
 };
+
+Room.prototype._savePendingEvents = async function() {
+    const pendingEvents = await Promise.all(
```

Comment:
Do we need this `await`/ `Promise` here? It seems like the inner part is sync...

Reply:
You're absolutely right, some left over when I was attempting to encrypt the messages there.